plugins/ScheduledReports/API.php

```diff
@@ -449,7 +449,7 @@ public function generateReport($idReport, $date, $language = false, $outputType
         // render report
         $description = str_replace(array("\r", "\n"), ' ', $report['description']);
 
-        list($reportSubject, $reportTitle) = self::getReportSubjectAndReportTitle(Site::getNameFor($idSite), $report['reports']);
+        list($reportSubject, $reportTitle) = self::getReportSubjectAndReportTitle(html_entity_decode(Site::getNameFor($idSite)), $report['reports']);
```

Comment:
@tsteur This part is to fix the double-escaping issues in scheduled reports (#7531)